client/src/components/UserAvatar/styles.css

```diff
@@ -0,0 +1,18 @@
+.user-avatar-container .user-avatar-spinner {
+    line-height: 100px;
+    margin-top: 30px;
+    text-align: center;
+    vertical-align: middle;
+}
+
+.user-avatar-container .avatar-picture {
+    text-align: center;
+    margin-top: 30px;
+    margin-bottom: 20px;
+}
+
+.user-avatar-container .avatar-picture img {
+    width: 200px;
+    box-shadow: 1px 3px 15px black;
+    border: solid 1px;
```

Comment:
Maybe it works, but correct variant is:
`border: 1px solid;`

Reply:
Thank you! I was thinking both ones were correct because both work fine... :D